forc-plugins/forc-doc/src/assets/sway.js

```diff
@@ -0,0 +1,185 @@
+/*
+Language: Sway
+Author: Fuel Labs contact@fuel.sh
+Contributors: Fuel Labs <contact@fuel.sh>,
+Website: https://fuel.sh
+Category: smart contracts, layer 2, blockchain,
+*/
```

Comment:
This is copied! Not sure how we would share it though since we can't rely on the user to have our `scripts` directory as well, I figured we _had_ to include it. Maybe there is another way though 🤔

Reply:
Whereabouts is it copied from?

If it's from the `sway-lsp` crate, perhaps we could put it in its own `sway-hljs` crate as a `const` included via `include_bytes!`? Then we could share that dependency between `forc-doc` and `sway-lsp`?